### PR TITLE
refactor / wallet reorg

### PR DESCRIPTION
**Pull request by yzhang1994**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**A description of the changes proposed in the pull request**:
- re-org files related to wallet and ethereum
- move wallet module to `hummingbot.wallet`

### CODE REVIEW

File: hummingbot/cli/hummingbot_application.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
 from wings.market.bamboo_relay_market import BambooRelayMarket
 from wings.order_book_tracker import OrderBookTrackerDataSourceType
 from wings.trade import Trade
-from wings.wallet.web3_wallet import Web3Wallet
+from wings.wallet.ethereum.web3_wallet import Web3Wallet
```
Comment (martinkou) [inline hummingbot/cli/hummingbot_application.py]: Why is this still referring to wings?

File: wings/wallet/ethereum/web3_wallet.pyx [hunk 2]
```diff
@@ -14,12 +14,12 @@ from hummingbot.core.clock cimport Clock
 from wings.events import (
     WalletEvent
 )
-from wings.ethereum_chain import EthereumChain
+from wings.wallet.ethereum.ethereum_chain import EthereumChain
```
Comment (martinkou) [inline wings/wallet/ethereum/web3_wallet.pyx]: It sounds like to me that this file should be deleted, not merely changed.

File: wings/wallet/ethereum/web3_wallet_backend.py [hunk 3]
```diff
@@ -0,0 +1,616 @@
+import asyncio
```
Comment (martinkou) [inline wings/wallet/ethereum/web3_wallet_backend.py]: Same here... we already have a copy in hummingbot right?

File: hummingbot/cli/hummingbot_application.py (hunk 1 above)
Comment (yzhang1994) [reply hummingbot/cli/hummingbot_application.py]: fixed

File: wings/wallet/ethereum/web3_wallet.pyx (hunk 2 above)
Comment (yzhang1994) [reply wings/wallet/ethereum/web3_wallet.pyx]: fixed

File: wings/wallet/ethereum/web3_wallet_backend.py (hunk 3 above)
Comment (yzhang1994) [reply wings/wallet/ethereum/web3_wallet_backend.py]: fixed

**Comment by martinkou** [comment]
Ok, I'll merge and bump hummingsim to fix the CI errors again.

**Pull request merged**